src/products/src/products-service/product.go

```diff
@@ -17,7 +17,14 @@ type Product struct {
 	Image          string  `json:"image" yaml:"image"`
 	Featured       string  `json:"featured,omitempty" yaml:"featured,omitempty"`
 	GenderAffinity string  `json:"gender_affinity,omitempty" yaml:"gender_affinity,omitempty"`
+	CurrentStock   int     `json:"currentstock" yaml:"currentstock"`
```

Comment:
JSON and YAML names should be `current_stock` for consistency with other multi-word attributes.